serverless.yaml

```diff
@@ -96,6 +100,18 @@ provider:
       Effect: Allow
       Resource:
         - '*'
+    - Action:
+        - 's3:ListBucket'

```

Comment:
Is this to allow creating presigned S3 urls? Is the ListBucket policy needed for creating presigned S3 urls?

Reply:
No, there's a role `ExportResultsSignerRole` that's used specifically for signing urls. We want to assume a different role so that we have control over the role session duration(otherwise the urls may stop working before the 30 min expiration time if the lambda role session expires before that).

The main lambda function needs `s3:ListBucket` to find all the result files for a given JobId

Reply:
Sounds good!